src/modules/auth/guards/auth.jwt-cognito.guard.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  ExecutionContext,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Reflector } from '@nestjs/core';
+import { AuthGuard } from '@nestjs/passport';
+import { User } from '@prisma/client';
+import { Observable } from 'rxjs';
+import { TOKEN_EXPIRED } from '../../../shared/constants/auth.constants';
+
+@Injectable()
+export class JwtCognitoAuthGuard extends AuthGuard('jwt-cognito') {
+  constructor(private reflector: Reflector) {
+    super(reflector);
+  }
+
+  canActivate(
+    context: ExecutionContext,
+  ): boolean | Promise<boolean> | Observable<boolean> {
+    const request = context.switchToHttp().getRequest();
```

Comment:
I forgot to delete this line here. We can leave just the super call or remove the methods entirely.